modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -33,14 +33,14 @@ The audit logs are not stored in the internal {product-title} Elasticsearch inst
 
 .Procedure
 
-To define index patterns and create visualizations:
+To define index patterns and create visualizations in Kibana:
 
-. Launch Kibana by clicking the Application Launcher {launch} and select *Logging*.
+. In the {product-title} console, click the Application Launcher {launch} and select *Logging*.
 
-. Create your Kibana index patterns:
+. Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
-** Regular users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
+** Users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
 
-** Admin users need to create index patterns for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 
+** Admin users must create index patterns for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 
```

Comment:
In line 42, "`app`" is formatted with `` I assume to indicate user text entry. In line 44, are "app", "infra", and "audit" also user-provided values? i.e., are users and admins doing the same process here, just with differences in what they create? 

Just struck me as a bit odd that they are formatted in bold instead, but that might be right based on the actual process for all I know! :)